### ISSUE TITLE
Do nothing with closed PRs

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt-pr-bot#38]
On https://github.com/web-platform-tests/wpt/pull/11122 @wpt-pr-bot changed the reviewers of a PR as I closed it. That generated email for me which wasn't very help. I suggest that when a PR is closed @wpt-pr-bot never does anything.

**Comment by foolip** [comment]
@jugglinmike do you think this was accidentally fixed by https://github.com/web-platform-tests/wpt-pr-bot/pull/73?

**Comment by foolip** [comment]
https://github.com/web-platform-tests/wpt-pr-bot/issues/69 is related.

I just saw on https://github.com/web-platform-tests/wpt/pull/13606 that this hasn't been fixed, the bot added back labels and reviewers that I removed before making a comment, thus spamming everyone.

**Comment by foolip** [comment]
This bug just caused a whole lot of spam in https://github.com/web-platform-tests/wpt/pull/13606 so I submitted https://github.com/web-platform-tests/wpt-pr-bot/issues/38.

@Ms2ger you asked there what I did but I won't reply there until this issue is fixed to avoid more spam. I just came along to try an understand the mishap, I didn't cause it. I can't pin down which force push it was, but probably one of the force pushes @woaiwyhty did on that PR either rebased a whole lot of history to new distinct commits, or contained a merge commit that had the same effect.
